lib/devices/light.js

```diff
@@ -108,41 +108,41 @@ module.exports = function(device, api) {
   }
 
   // On realtime event update the device
-  device.on('$capabilitiesObj', debounce(capabilitiesObj => {
+  device.on('$update', id => {
```

Comment:
Geen `debounce` meer hier?

Reply:
had een beetje aangenomen dat dit al in het update event werd gedaan. Uit de documentatie

$update
Fires when this instance has been updated.

Reply:
Uit die documentatie haal ik niet dat het event gedebounced wordt. Stel dat je de kleur van een lamp verandert, enig idee of je dan één `$update` event krijgt, of meerdere voor elke relevante capability (`hue`, `saturation`, etc)?

Reply:
net even getest. Je krijgt maar 1 update event. Maar het duurt wel een seconde of 5 voordat je deze krijgt

Reply:
Pff 5 seconden, dat is best lang. Klinkt alsof ze nu zelf een debounce doen.